### DISCUSSION TITLE
Improvement to User Interface of Crowdfunding Pages

### DISCUSSION THREAD

**Discussion by andysavage** [btcpayserver/btcpayserver#5376] [Ideas / Feature Requests]
Hi I am working on a crowdfunding page, I don't want to offer perks, but the perks still appear when hovering over the payment button (donate custom amount). Is there a way to get rid of this? It's probably confusing to visitors who want to donate. 

Needs to either change the message or perhaps remove the sidebar completely if there is no perks added. 

It can be seen on [this page](https://btcpay.satoshihost.com/apps/2teypGodqcD16tRNtsUbMq8FSBQA/crowdfund), if you hover over "donate custom amount".

**Comment by dennisreimann** [comment]
Agreed, will fix this :)

**Reply by pavlenex** [reply]
Is this still an issue?

**Reply by andysavage** [reply]
Hi and thanks for replying!

 I just clicked to update to make sure it's on the latest version and then tried it again and it does still say "Select this contribution perk". I attach 2 images - the cursor doesn't show in my screenshots but both are on hover. One on the sidebar "Donate Custom Amount" button and one on the modal that appears when you click the main "Contribute" button.

![sidebar on hover](https://github.com/btcpayserver/btcpayserver/assets/12832713/af6a3611-ca57-40df-8691-67ba28c71306)
![modal on hover](https://github.com/btcpayserver/btcpayserver/assets/12832713/ea7677b7-7be8-45b6-9b64-c50caab20f65)

**Reply by andysavage** [reply]
I should add I have closed that fundraiser because we got enough to do the projects so it will not work now. But it should be present in any other instance too.
